docs/guides/quickfort-alias-guide.rst

```diff
@@ -499,17 +517,23 @@ give10up
 give10down
 give10left
 give10right
-give             move
+give              move
 togglesequence
 togglesequence2
-===============  ===========
-
-``linksonly`` and ``nocontainers`` set the named basic properties on
-stockpiles. ``nocontainers`` sets bins and barrels to 0, but does not affect
-wheelbarrows since the hotkeys for changing the number of wheelbarrows depend
-on whether you have the DFHack `stockpiles` plugin active. It is better to set
-the number of wheelbarrows via the `quickfort` ``stockpiles_max_wheelbarrows``
-setting. It is set to ``0`` by default.
+masterworkonly    prefix
+artifactonly      prefix
+togglemasterwork  prefix
+toggleartifact    prefix
+================  ===========
+
+``linksonly``, ``maxbins``, ``maxbarrels``, ``nobins``, ``nobarrels``, and
+``nocontainers`` set the named basic properties on stockpiles. ``nocontainers``
+sets bins and barrels to 0, but does not affect wheelbarrows since the hotkeys
+for changing the number of wheelbarrows depend on whether you have the DFHack
+`stockpiles` plugin active. It is better to set the number of wheelbarrows via
```

Comment:
It's `tweak max-wheelbarrow` that changes the UI, actually

Reply:
whoops! fixed. Thanks!